### DISCUSSION TITLE
How to fix weird fading on left on mobile

### DISCUSSION THREAD

**Discussion by SamsungGalaxyPlayer** [upptime/upptime#640] [Q&A]
On mobile (and on very narrow desktop browsers), the entries fade on the left side as shown below. Is there any way to disable this?

![image](https://user-images.githubusercontent.com/12520755/185634020-5283185b-dd9f-41b5-9f78-fcb068874cef.png)

**Comment by AnandChowdhary** [comment]
Indeed very weird! The https://github.com/upptime/status-page repository has this and we'd need to look for a gradient somewhere in the source code that's leaking out.
